packages/components/datepicker/src/Datepicker.tsx

```diff
@@ -1,27 +1,184 @@
-import React from 'react';
+import React, {
+  ChangeEventHandler,
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
 import { cx } from 'emotion';
 import { CommonProps } from '@contentful/f36-core';
 
+import { format, isValid, parse, startOfDay, endOfDay } from 'date-fns';
 import { getStyles } from './Datepicker.styles';
 import { Calendar } from './Calendar';
+import { Popover } from '@contentful/f36-popover';
+import { DayPickerSingleProps } from 'react-day-picker';
+import FocusLock from 'react-focus-lock';
+import { TextInput, TextInputProps } from '@contentful/f36-forms';
+import { IconButton } from '@contentful/f36-button';
+import { CalendarIcon } from '@contentful/f36-icons';
+import { PopoverProps } from '@contentful/f36-popover/src';
 
-export type DatepickerProps = CommonProps;
+const DATE_FORMAT = 'dd/MM/yyyy';
 
-function _Datepicker(props: DatepickerProps, ref: React.Ref<HTMLDivElement>) {
+// Props comparison to our current datepickers:
+// Current      DayPicker
+// value        selected
+// onChange     onSelect
+// min          fromDate
+// max          toDate
```

Comment:
Is there a reason to keep this in both the README and here? I prefer README, so we don't have to update/remove two places.